cocos/network/CCDownloader-curl.cpp

```diff
@@ -845,7 +847,7 @@ namespace cocos2d { namespace network {
             DLLOG("    DownloaderCURL: finish Task: Id(%d)", coTask.serialId);
         }
         
-        if (_impl->stoped())
+        if (!_breakScheduleFunc && _impl->stoped())
```

Comment:
When the current object became wild pointer, we can't assume its property value remains the same, so there may still be problems